clearpath_generator_common/clearpath_generator_common/param/platform.py

```diff
@@ -236,6 +236,28 @@ def generate_parameters(self, use_sim_time: bool = False) -> None:
                                     ],
                                     'contains': ['MCU']}}}}})
 
+            # Add cooling for A300 only for now
+            if self.clearpath_config.get_platform_model() == Platform.A300:
+                self.param_file.update(
+                    {self.DIAGNOSTIC_AGGREGATOR_NODE: {
```

Comment:
Can we please keep opening braces either:
- at the end of the previous line (`self.param_file.update({`), or
- on their own line with the contents indented?